src/main/java/core/gui/comp/table/PlayersTable.kt

```diff
@@ -0,0 +1,87 @@
+package core.gui.comp.table
+
+import core.model.HOVerwaltung
+import core.model.player.Player
+import java.beans.PropertyChangeEvent
+import java.beans.PropertyChangeListener
+import java.beans.PropertyChangeSupport
+import javax.swing.event.ListSelectionEvent
+
+class PlayersTable @JvmOverloads constructor(tableModel: HOTableModel, fixedColumnsCount: Int = 1) :
+    FixedColumnsTable(tableModel, fixedColumnsCount), PropertyChangeListener {
+
+        companion object {
+
+            private val propertyChangeSupport : PropertyChangeSupport = PropertyChangeSupport(this)
+
+            fun addPropertyChangeListener(listener: PropertyChangeListener?) {
+                propertyChangeSupport.addPropertyChangeListener(listener)
+            }
+
+            fun removePropertyChangeListener(listener: PropertyChangeListener?) {
+                propertyChangeSupport.removePropertyChangeListener(listener)
+            }
+
+            fun firePropertyChanged(oldSelection: Player?, newSelection: Player?){
+                propertyChangeSupport.firePropertyChange("SelectedPlayer", oldSelection, newSelection)
+            }
+
+            private var selectedPlayer: Player? = null
+
+            fun setSelectedPlayer(player: Player?){
+                if ( player != selectedPlayer){
+                    val oldSelection = selectedPlayer
+                    selectedPlayer = player
+                    firePropertyChanged(oldSelection, player)
+                }
+            }
+
+            fun getSelectedPlayer() : Player?{
+                return selectedPlayer
+            }
+        }
```

Comment:
Those companion object methods bother me, but I think this would be too much work to fix at this point.  We can add a TODO to re-visit.

Reply:
It is my approach to replace the PlayerSelectionListener. I want all player tables in HO to inherit from this class to synchronize their player selection. Now these are the training tables. In a next step the player overview should be added. Then selecting a player in any table will select the same player in each other player table.